### PR TITLE
tests: Migrate member list tests to common framework

### PR DESCRIPTION
**Pull request by clarkfw**
Context #13637 

Signed-off-by: Clark <fwyongxing@gmail.com>

### CODE REVIEW

**Comment by serathius** [comment]
I would recommend to not do everything in one PR. If you can migrate one test independently it would be faster for us to iterate.

**Comment by clarkfw** [comment]
> I would recommend to not do everything in one PR. If you can migrate one test independently it would be faster for us to iterate.

sounds good. 
I've made some changes: In this PR, I'll only migrate member list tests.

File: tests/common/member_test.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
+// Copyright 2016 The etcd Authors
```
Comment (serathius) [inline tests/common/member_test.go:1]:
```suggestion
// Copyright 2022 The etcd Authors
```

**Review by serathius — APPROVED** [review]
This PR makes me super excited. With only couple of lines the test is much more readable as we reuse so much code.

For context this is the old test:
```
func ctlV3MemberList(cx ctlCtx) error {
	cmdArgs := append(cx.PrefixArgs(), "member", "list")
	lines := make([]string, cx.cfg.ClusterSize)
	for i := range lines {
		lines[i] = "started"
	}
	return e2e.SpawnWithExpects(cmdArgs, cx.envMap, lines...)
}
```

**Comment by serathius** [comment]
DCO failed after adding the suggestion commit. Can you squash the commits in PR?

**Pull request merged**